std_execution.bs

```diff
@@ -1098,6 +1098,24 @@ execution::sender auto snd = execution::then(vals, [](auto... args) {
 
 This adaptor is included as it greatly simplifies lifting values into senders.
 
+### `execution::just_error` ### {#design-sender-factory-just_error}
+
+<pre highlight="c++">
+execution::sender auto just_error(
+    auto && error
+);
+</pre>
+
+Returns a sender with no [=completion scheduler|completion schedulers=], which completes with the specified error. If the provided error is an lvalue reference, a copy is made inside the returned sender and a non-const lvalue reference to the copy is sent to the receiver's `set_error`. If the provided value is an rvalue reference, it is moved into the returned sender and an rvalue reference to it is sent to the receiver's `set_error`.
```

Comment:
I don't understand why we'd pass an lvalue-reference if we have already copied the value.

Could this be simplified to just say that `just_error(e)` decay-copies `e` and sends an rvalue-reference to the copied value when started?

Reply:
This is for consistency with the specification of `just`. I don't particularly care for this behavior either, and IMO we should revisit this for both `just` and `just_error` in R3.